src/main/webapp/app/modules/administration/client-management/client-management.reducer.ts

```diff
@@ -129,6 +181,13 @@ export const updateClient: ICrudPutAction<IClient> = client => async dispatch =>
   const result = await dispatch({
     type: ACTION_TYPES.UPDATE_CLIENT,
     payload: axios.put(clientApiUrl, client)
+  }).then(res => {
+    dispatch(
+      updateClientProfile({
```

Comment:
We shouldn't depend on UI when saving client profiles - what if only one of the request fails - the auth and core services would get out of sync. The core service should make a request on behalf of the user and save the client along with the profile